src/utils_lua.c

```diff
@@ -24,10 +24,7 @@
  *   Florian Forster <octo at collectd.org>
  **/
 
-/* <lua5.1/luaconf.h> defines a macro using "sprintf". Although not used here,
- * GCC will complain about the macro definition. */
-#define DONT_POISON_SPRINTF_YET
-
+#include "common.h"
```

Comment:
Same here. `common.h` is already included 2 lines below. Let's remove this line.